src/lib/airtable/schema.js

```diff
@@ -1,6 +1,6 @@
 /*
-    THIS IS A GENERATED FILE
-    Changes might be overwritten in the future, edit with caution!
+	THIS IS A GENERATED FILE
```

Comment:
Can you revert the formatting changes made here? We don't want to change the generated file apart from apply patches, or else we risk patch conflicts (because they depend on diffs).